### PR TITLE
Do not clear CommentForm on upvote

### LINKED ISSUE #807: Comment form should not be cleared on upvote
comment-vote

#### Expected behavior

The content in the comment form should stay untouched after upvoting the post.

#### Actual behavior

Comment form is cleared after upvoting the post.

#### How to reproduce

1. Start writing a comment for a post. Don't submit it yet.
2. Upvote the post.
3. The comment form is cleared.

* Browser: Chrome 61.0.3163.100
* Operating system: Windows 10

### PR DESCRIPTION
**Pull request by jkarthionline**
Fixes #807 

Changes:
- CommentForm Component inputValue is cleared while doing the upvote. The changes has been made to stop that

### CODE REVIEW

**Comment by bonustrack** [comment]
Thank you for your contribution, it seem to be working. I have 2 changes for you, 
1: Can you fix ESLint errors see https://circleci.com/gh/busyorg/busy/2143?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
```
 55:55  error  A space is required before '}'  object-curly-spacing
  189:37  error  A space is required before '}'  object-curly-spacing
```
2: When i click on comment button, the input seem cleared:
image

While it should show the actual comment like this: (from busy.org)
image

File: src/components/Comments/CommentForm.js [hunk 1]
```diff
@@ -51,8 +53,10 @@ class CommentForm extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!nextProps.isLoading) {
-      this.setState({ inputValue: nextProps.inputValue || '' });
+    if (!nextProps.isLoading && nextProps.inputValue !== '') {
```
Comment (bonustrack) [inline src/components/Comments/CommentForm.js]: You could do this instead 
```
if ((!nextProps.isLoading && nextProps.inputValue !== '') || nextProps.commentSubmitted) {
```
That would make the code shorter as we dont need the next lines duplicated.

**Comment by jkarthionline** [comment]
@bonustrack Shorten the code as per your suggestion

File: src/components/Comments/CommentForm.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ class CommentForm extends React.Component {
     onImageInserted: PropTypes.func,
     onImageInvalid: PropTypes.func,
     onSubmit: PropTypes.func,
+    commentSubmitted: PropTypes.bool,
```
Comment (Sekhmet) [inline src/components/Comments/CommentForm.js]: We usually have `func` props at the bottom. Can you move this prop up? I think it makes sense to rename this props to `submitted`.

File: src/components/Comments/CommentForm.js [hunk 3]
```diff
@@ -31,6 +32,7 @@ class CommentForm extends React.Component {
     onImageInserted: () => {},
     onImageInvalid: () => {},
     onSubmit: () => {},
+    commentSubmitted: false,
```
Comment (Sekhmet) [inline src/components/Comments/CommentForm.js]: This should be moved up as well.

**Pull request merged**